Doc/whatsnew/3.15.rst

```diff
@@ -533,6 +533,12 @@ http.cookies
 * Allow '``"``' double quotes in cookie values.
   (Contributed by Nick Burns and Senthil Kumaran in :gh:`92936`.)
 
+idle
+----
+
+*   Add a "Reload from Disk" item to the File menu. This allows discarding
+    unsaved changes and reloading the current version of the file from the disk.
+    (Contributed by Shamil Abdulaev in :gh:`44968`.)
```

Comment:
```suggestion


idlelib
-------

*   Add a "Reload from Disk" item to the File menu. This allows discarding
    unsaved changes and reloading the current version of the file from the disk.
    (Contributed by Shamil Abdulaev in :gh:`44968`.)

```

The module is idlelib but I'm not sure we need to document the module itself. Can you check in previous What's New files (3.12, 3.13 or older) where we documented such changes please?